corehq/apps/users/signals.py

```diff
@@ -29,13 +29,7 @@ def _update_user_in_es(couch_user):
     if couch_user.to_be_deleted():
         user_adapter.delete(couch_user.user_id)
     else:
-        user_adapter.index(_transform_for_es(couch_user))
-
-
-def _transform_for_es(couch_user):
-    """Implemented separately for test patching."""
```

Comment:
FYI, [this test](https://github.com/dimagi/commcare-hq/blob/3f4877a8d31/corehq/apps/users/tests/test_signals.py#L134) also needs to be updated (remove the `simple_doc` function and `with patch(...)` block).